### DISCUSSION TITLE
desktop Coinbase extension connects but mobile Coinbase does not connect. It redirects to the website but Meta Mask is working

### DISCUSSION THREAD

**Discussion by ripon20244** [WalletConnect/walletconnect-monorepo#4138] [Developer Support] [answered]
desktop Coinbase extension connects but mobile Coinbase does not connect. It redirects to the website but Meta Mask is working

**Comment by 1790905940** [comment]
Right

**Comment by 1790905940** [comment]
Right

**Comment by glitch-txs** [comment, marked as answer]
this is how Coinbase SDK works at the moment, there's a beta feature we might add but this will be added in Wagmi v2 and ethers v6

**Reply by wpcorbett79** [reply]
On the previous version of Web3 Modal, Coinbase mobile connect automatically opened a browser within Coinbase wallet. Why had this been changed? It was working fine but now does not work at all. Also the CB QR code on desktop extension opens to MetaMask.
